bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet.java

```diff
@@ -511,7 +521,9 @@ private EnhancedRendition getBestRendition(@NotNull Asset asset, int width) thro
     @NotNull
     private EnhancedRendition getOriginal(@NotNull Asset asset) throws IOException {
         EnhancedRendition original = new EnhancedRendition(asset.getOriginal());
-        return filter(original);
+        EnhancedRendition filtered = filter(original);
+        metrics.markOriginalRenditionUsed();
```

Comment:
`getOriginal` is only triggered when the original renditions is requested explicitly. It could also be returned by `getBestRendition` implicitely (in the `for.. if` loop).